DualityEditor/Utility/Extensions/ExtMethodsDualityFont.cs

```diff
@@ -36,47 +36,53 @@ public static class ExtMethodsDualityFont
 		/// </summary>
 		public static void RenderGlyphs(this DualityFont font, FontRenderGlyphCharSet extendedSet)
 		{
-			if (font.EmbeddedTrueTypeFont == null) throw new InvalidOperationException("Can't render glyphs of a Duality Font without embedded vector Font information.");
```

Comment:
This was changed purely because changing properties on a Font would make the preview fail since it's waiting on the EmbeddedFont. My thinking is with a bitmap font you'd want to recreate kernings (currently since they are always used if the arrays are set) only, while with a bitmap font, you might have changed size which should re-generate all the glyphs.